core/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactorySpec.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the AnswerClassificationResultObjectFactory.
+ */
+ 
+describe('Answer classification result object factory', function() {
+  var AnswerClassificationResultObjectFactory;
+  var sampleOutcome = {
```

Comment:
In AnswerClassificationResultObjectFactory.createNew(), is the first arg supposed to be an Outcome domain object (i.e. an instance of what's created by OutcomeObjectFactory)? If so, that's different from this, which is an outcomeDict.

Reply:
Oh right! Just got little confused with line 370 in `AnswerClassificationServiceSpec.js`, I thought it can just be a dict.
@jervis446, you need to inject `OutcomeObjectFactory` and do something similar to line 381 in `AnswerClassificationServiceSpec.js`.

Reply:
@shubha1593 you mean like something similar to `oof = $injector.get('OutcomeObjectFactory');`
and first arg int createNew is like `oof.createNew('outcome 1', '', [])` ? because in `AnswerClassificationServiceSpec.js` there is no 381th line.